jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettings.kt

```diff
@@ -107,7 +121,9 @@ data class AwsConfiguration(
     var isTelemetryEnabled: Boolean? = null,
     var promptedForTelemetry: Boolean? = null,
     var useDefaultCredentialRegion: String? = null,
-    var profilesNotification: String? = null
+    var profilesNotification: String? = null,
+    var isAutoUpdateEnabled: Boolean? = null,
+    var isAutoUpdateNotificationDoNotShowAgain: Boolean? = null
```

Comment:
umm, maybe we should consider if we can re-use this API https://github.com/JetBrains/intellij-community/blob/idea/232.10203.10/platform/ide-core/src/com/intellij/notification/Notification.java#L160

Reply:
will take a look

Reply:
I don't see any existing usage of this API, plus this seems to only apply to the Notification object itself, which notifyInfo() doesn't return any.